communication.rst

```diff
@@ -15,9 +15,9 @@ python-dev_ is the primary mailing list for discussions about Python's
 development. The list is open to the public and is subscribed to by all core
 developers plus many people simply interested in following Python's
 development. Discussion is focused on issues related to Python's development,
-such as how to handle a specific issue, a PEP, etc.::
+such as how to handle a specific issue, a PEP, etc.
 
-   - Ideas about new functionality should **not** start here and instead
+   - Ideas about new functionality should **NOT** start here and instead
```

Comment:
Grey background is added by GitHub. I was trying to show an example in reST markup. See https://guides.github.com/features/mastering-markdown/#examples

What I was trying to say is that whitespaces in the beginning of the lines can be deleted:

Change

```rst
such as how to handle a specific issue, a PEP, etc.

   - Ideas about new functionality should **not** start here and instead
     should be sent to python-ideas_.
```

to

```rst
such as how to handle a specific issue, a PEP, etc.

- Ideas about new functionality should **not** start here and instead
  should be sent to python-ideas_.
```